kernel-r/declarativewidgets/R/serializers.r

```diff
@@ -1,5 +1,14 @@
 #' @include serializer.r
 
+serialize_element <- function(elem) {
+    if(class(elem) == "Date") {
+        options(digits.secs=3)
+        return (format(elem, "%Y-%m-%dT%H:%M:%OSZ"))
```

Comment:
I see what you mean regarding using a default if the function is not provided a format.  In python for example that exists, but its exposed internally and not used [example](https://github.com/jupyter-incubator/declarativewidgets/blob/master/kernel-python/urth/widgets/widget_dataframe.py#L53).  It may be less convoluted for the consumer to try to deal with one global type or maybe not once the broader example is in place.